server/syntax_gen.js

```diff
@@ -0,0 +1,77 @@
+module.exports = function () {
+	var fs = require('fs');
+	var path = require('path');
+	var nodes = require('./nodetypes/nodetypes.js');
+
+	var file_path = path.join(__dirname, '../client/libs/ace-builds/src-min/snippets/fbp.js');
```

Comment:
Can ace load a snippet inside of the folder ```soletta-dev-app/client/js/ace``` ? 

This would be better if a new ace version changes the names of the directory, we would avoid some problems.

Also, the folder libs should be changed only by bower.